### DISCUSSION TITLE
Deprecation notice

### DISCUSSION THREAD

**Discussion by RishabhKarnad** [GeekyAnts/vue-native-core#342] [General]
Hello everyone,

We are officially announcing that Vue Native will no longer be maintained and that it is being deprecated.

If you have been using Vue Native, please consider migrating to React Native instead, since it will allow you to continue using the React Native ecosystem
https://reactnative.dev/

If you wish to continue using Vue for mobile app development, please consider using NativeScript-Vue or Capacitor instead. Both of these frameworks will allow you to upgrade to Vue 3 as well if you need.
https://v3.vuejs.org/guide/mobile.html

**Comment by arokeaugustine** [comment]
Please why was Vue native deprecated? I just got to know about it and mhen I love the syntax. It's rather unfortunate it's already deprecated 😔.

**Comment by osh91** [comment]
MAKE IT ALIVE >:( DON'T GIVE UP.. JUST DOOO IT
MAKE VUE NATIVE AGAIN! WE NEED YOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOUUUU :(

**Reply by osh91** [reply]
Well, I went to Dart Side. 
_Too late_

**Reply by feyfath** [reply]
how is the dart side going so far

**Reply by osh91** [reply]
Hi there!
_Coming to you with loads of bird-sht from Dash_  😒

_Uhm… well… ☝🏼✊🏼..._

For **mobile phones**? Sure, no problem! 📱
For **desktop and web**? Absolutely not! 🚫💻

**Shall I stick with React Native**?
Hell no—please, I **beg** you! 🙏🏼🔥

What about **KMM** (Kotlin Multiplatform Mobile)?
I haven’t tried it yet, but I never will. Why? 🤔
Because I don’t think it’s smart to go with cross-platform frameworks—**they’re all crap**. 💩

So you’re saying **I should create separate apps**, like one for Android using Jetpack and one for iOS with XCode?
Yes! A strong yes! 💪🏼

But that will cost more… right?
Yes and no. You’ll build apps with better quality than anything cross-platform can deliver. 🏆✨

____

FYI: Dash is Flutter’s mascot. If you make even the tiniest mistake, Dash will rain down on you with a storm of errors—aka bird-sh*t. 🐦💩

**Comment by VRNeves** [comment]
Please, don't give up. 
Don't make me cry.
Don't say goodbye...
Don't run around...

**Comment by Oscar-Raygoza** [comment]
😢

**Comment by uriboshogo** [comment]
worst news of the year

**Comment by alsawaqi** [comment]
what is the best alternative other than react native

**Reply by Oscar-Raygoza** [reply]
https://ionicframework.com/docs/ 
https://quasar.dev/video-tutorials
https://flutter.dev/

**Reply by osh91** [reply]
Go flutter  🐦

**Comment by monoira** [comment]
dude...

**Reply by angelhdzmultimedia** [reply]
IonicVue, Quasar + Capacitor, Nuxt 3 + Capacitor.

**Comment by rxfr-dvpr** [comment]


**Reply by osh91** [reply]
hahahahahah!!!!
Hey Have you tried Flutter?? :) Try!
